setup.py

```diff
@@ -20,7 +20,7 @@
         "pytest-cov>=2.8.0, <3.0.0",
         "tox==3.21.1",
         "pre-commit==2.9.3",
-        "apache-airflow==2.0.1",
+        "apache-airflow[kubernetes]==2.0.1",
```

Comment:
Is `kubernetes` required here? I don't see new imports used during tests.

Reply:
It is imported in the `node_pod`
https://github.com/getindata/kedro-airflow-k8s/pull/71/files#diff-b84725b0af420af10bbf6701b8616b8bc88b57ea4708675a864f651ce9e9eaa9R10
Which is used in `tests/operators/test_node_pod_operator.py`

Reply:
That's true, it's just that we had this test in the past and it was working correctly without `kubernetes` extra for apache-airflow. If it's removed, the tests should still work, shouldn't they?

Reply:
I am importing it for the `Secret` type annotation. Without this entry in `setup.py` the test was failing.